code/modules/power/cell.dm

```diff
@@ -44,6 +45,9 @@
 		STOP_PROCESSING(SSobj, src)
 	return ..()
 
+/obj/item/cell/get_rating()
+	return rating
```

Comment:
wouldnt it make sense to just use the maxcharge var instead of an extra new var?